front/pages/events_map.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Upcoming events map
+ */
+export default (mapFactory, api) => {
+    api.getUpcomingEvents((events) => {
+        const map = mapFactory.createMap(dom('#map'), {
+            center: { lat: 46.7699, lng: 2.4279 },
```

Comment:
I guess so! I just copied/pasted them from the JavaScript code to display the map of committees.